packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -41,7 +42,11 @@ export interface IDetailsList extends IList {
   getStartItemIndexInView: () => number;
 }
 
-export interface IDetailsListProps extends React.Props<DetailsList>, IWithViewportProps {
+export interface IDetailsListProps extends React.Props<DetailsListBase>, IWithViewportProps {
+  theme?: ITheme;
+
+  styles?: IStyleFunctionOrObject<IDetailsListStyleProps, IDetailsListStyles>;
```

Comment:
Can we please document both styles and themes? (I'm sure you can find examples of components which do not, but most do - and we really ought to be documenting)